python/fledge/services/core/api/plugins/remove.py

```diff
@@ -360,8 +360,9 @@ async def _check_plugin_usage_in_notification_instances(plugin_name: str) -> lis
     return notification_instances
 
 
-async def _put_refresh_cache(protocol: str, host: int, port: int) -> None:
-    management_api_url = '{}://{}:{}/fledge/cache'.format(protocol, host, port)
+async def _put_refresh_cache(host: int, port: int) -> None:
```

Comment:
I would suggest to keep these def as is, and from caller pass http only. We may really want to run all this on TLS in near future.